packages/react-i18n/src/test/manager.test.ts

```diff
@@ -22,11 +22,11 @@ describe('Manager', () => {
     switch (locale) {
       case 'en':
         return enParent;
-      case 'en-us':
+      case 'en-US':
```

Comment:
Isn't this going to break the `requests translations for the (lowercase) full locale and language when the country code is provided` test?

Reply:
I can actually leave it as is, both should work. I've updated the function for it to return:
`["pt-BR", "pt-br", ""pt"]` so everything would work.

Reply:
I've updated some tests after all.